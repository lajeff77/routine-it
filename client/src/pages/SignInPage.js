import React from 'react';
import MainHeader from '../components/MainHeader';
import Footer from '../components/Footer';

const SignInPage = () => {

    return (
        <div>
            <MainHeader />
            <Footer />
        </div>
    )
}

export default SignInPage;