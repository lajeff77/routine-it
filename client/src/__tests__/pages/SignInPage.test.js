import { render} from "@testing-library/react";
import React from "react";
import { MemoryRouter, Route, Routes } from "react-router-dom";
import SignInPage from "../../pages/SignInPage";


describe(SignInPage.name, () => { 
    test('SignInPage matches snapshot', () => {
        const { asFragment } = render(
            <MemoryRouter>
            <Routes>
                <Route element={<SignInPage />} path="/signin" />
            </Routes>
            </MemoryRouter>
        );
    
        expect(asFragment()).toMatchSnapshot();
    });
});