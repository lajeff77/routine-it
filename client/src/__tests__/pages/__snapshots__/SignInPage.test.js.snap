// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInPage SignInPage matches snapshot 1`] = `<DocumentFragment />`;
